test/language/module-code/comment-single-line-html-close-comment-newline-before-function.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: >
+    HTML-like comments are not available in module code
+    (SingleLineHTMLCloseComment)
+esid: sec-html-like-comments
+es6id: B1.3
+negative:
+  phase: early
+  type: SyntaxError
+flags: [module]
+---*/
+-->
+function f(){}
```

Comment:
I'm not sure what I'm missing here... what's different from the previous file? Is it just the empty line that precedes `-->`?